playground/README.md

```diff
@@ -15,7 +15,7 @@ Compile bdk-cli using the right features and the WASM target:
 
 ```
 cd bdk-cli
-RUSTFLAGS="-C opt-level=s -C strip=symbols" cargo build --release --no-default-features --features esplora-reqwest,async-interface,compiler --target wasm32-unknown-unknown
+RUSTFLAGS="-C opt-level=s -C strip=symbols" cargo build --release --target wasm32-unknown-unknown --no-default-features --features esplora-reqwest,async-interface,compiler,dev-getrandom-wasm
 ```
 
```

Comment:
Can we mention something like this too in the README too?
```suggestion

If building on Mac M1, due to bitcoin/rust-secp256k1#283 you  must add to the build command `AR=/opt/homebrew/opt/llvm/bin/llvm-ar CC=/opt/homebrew/opt/llvm/bin/clang`.
```

Reply:
Updated, thanks!